### ISSUE TITLE
Error installing and opening SimBA

### ISSUE THREAD

**Issue by fon215** [sgoldenlab/simba#186]
**Describe the bug**

I am installing SimBA on my computer to be used with DeepLabCut to perform automated behavioral analysis with mice to study pain and itch behaviors for a lab. I have encountered some problems during the installation process. First of all, I am not able to extract the files in FFmpeg in step 6 of the directions for installing FFmpeg. Could you let me know how to complete these steps and extract the files? I have included screenshots showing this as well.

Also, I am having trouble launching simba once I complete the installation steps.

I have highlighted here the command line with the prompts I used and the errors I received. Could you help me figure out what I am doing wrong so that I am able to install the program? I will also include a screenshot of the steps for installation that I was following.
____________________________________________________________

(base) C:\Users\maf315>conda activate simbaenv

(simbaenv) C:\Users\maf315>pip install pip==19.0.1
Collecting pip==19.0.1
  Using cached pip-19.0.1-py2.py3-none-any.whl (1.4 MB)
Installing collected packages: pip
  Attempting uninstall: pip
    Found existing installation: pip 21.2.2
    Uninstalling pip-21.2.2:
      Successfully uninstalled pip-21.2.2
ERROR: Could not install packages due to an OSError: [WinError 5] Access is denied: 'C:\\Users\\maf315\\AppData\\Local\\Temp\\pip-uninstall-g4k_1xih\\pip.exe'
Consider using the `--user` option or check the permissions.

(simbaenv) C:\Users\maf315>pip install simba-uw-tf-dev
Collecting simba-uw-tf-dev
  Using cached https://files.pythonhosted.org/packages/e7/b5/4b8d0476ae338167a394fa04e799b198404ea182401748c1bdc2e4387dd5/Simba_UW_tf_dev-0.91.9-py3-none-any.whl
Collecting dash==1.14.0 (from simba-uw-tf-dev)
Collecting pyarrow==0.17.1 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/09/b4/4d361463cb3fbaddb0292d8cafc7134e43e66f82144db0c4cb433dac2624/pyarrow-0.17.1-cp36-cp36m-win_amd64.whl
Collecting Pillow==5.4.1 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/ec/ca/7af5b6628ecf770645f8cc3c9da3c2bb5c5ffc7384a9ff0666fdb818b4d5/Pillow-5.4.1-cp36-cp36m-win_amd64.whl
Collecting wxpython==4.0.4 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/8f/94/17dc5e0191351e2d7a7f9b79eb355de930feade9c7decec70b19e0f2e160/wxPython-4.0.4-cp36-cp36m-win_amd64.whl
Collecting opencv-python==3.4.5.20 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/39/7e/5578b4f6f7d5b7b8387d4d3de4fd414d75cf45701a5ab7a23e656e865417/opencv_python-3.4.5.20-cp36-cp36m-win_amd64.whl
Collecting tqdm==4.30.0 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/76/4c/103a4d3415dafc1ddfe6a6624333971756e2d3dd8c6dc0f520152855f040/tqdm-4.30.0-py2.py3-none-any.whl
Collecting numpy==1.18.1 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/53/74/b997e4c7b4abc668e99f4c3dba87ee2c6f7559319af756cc1ede37665a8d/numpy-1.18.1-cp36-cp36m-win_amd64.whl
Collecting tabulate==0.8.3 (from simba-uw-tf-dev)
Collecting numba==0.48.0 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/ae/61/15ac002290c77694aa88aa9ead95879e7121b9ad40b23f9b89701bec355b/numba-0.48.0-cp36-cp36m-win_amd64.whl
Collecting numexpr==2.6.9 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/f7/2d/ed626045f9e4c17fb6c2dea95a50b6e484427e5d324b6a0c4c53be1a4ddb/numexpr-2.6.9-cp36-none-win_amd64.whl
Collecting pyyaml==5.3.1 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/61/5e/0386f8d74ce6e0a183b93a7e02d32b2629c0a2c8e9db806a3f5eebbfa221/PyYAML-5.3.1-cp36-cp36m-win_amd64.whl
Collecting yellowbrick==0.9.1 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/d8/e8/125204ea84a7424a3237556e8dfaec9fee21f2e3d5b3695eb9ce355bf668/yellowbrick-0.9.1-py2.py3-none-any.whl
Collecting eli5==0.10.1 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/97/2f/c85c7d8f8548e460829971785347e14e45fa5c6617da374711dec8cb38cc/eli5-0.10.1-py2.py3-none-any.whl
Collecting pandas==0.25.3 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/f0/ac/92c3d2f0b627efbd1a7b2156faa697f9c2bbd7b0fe83ba8a9d36f982156f/pandas-0.25.3-cp36-cp36m-win_amd64.whl
Collecting dash-color-picker==0.0.1 (from simba-uw-tf-dev)
Collecting imblearn==0.0 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/81/a7/4179e6ebfd654bd0eac0b9c06125b8b4c96a9d0a8ff9e9507eb2a26d2d7e/imblearn-0.0-py2.py3-none-any.whl
Collecting scipy==1.1.0 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/62/e2/364f0bcc641aeff79d743c732769d5dc31a1e78c27699229431412c4b425/scipy-1.1.0-cp36-none-win_amd64.whl
Collecting scikit-learn==0.22.2 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/30/ef/528c6ccf1986bab9e9a3e53e0f1e673ed5f1dd348dc32618c670b61100f6/scikit_learn-0.22.2-cp36-cp36m-win_amd64.whl
Collecting graphviz==0.11 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/af/ae/e1c63ac4c531d69a7960a99af99e184d4f3da15e29f67767c4252bf19cce/graphviz-0.11-py2.py3-none-any.whl
Collecting h5py==2.9.0 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/01/1e/115c4403544a91001d9c618748b2e8786db45544e36b8a6cf3c525e9b57f/h5py-2.9.0-cp36-cp36m-win_amd64.whl
Collecting xgboost==0.90 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/5e/49/b95c037b717b4ceadc76b6e164603471225c27052d1611d5a2e832757945/xgboost-0.90-py2.py3-none-win_amd64.whl
Collecting statsmodels==0.9.0 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/77/2b/8ba61399b31f984c263b177c2e2547a34f0d4d972a24a51fc77c376079b0/statsmodels-0.9.0-cp36-cp36m-win_amd64.whl
Collecting plotly==4.9.0 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/bf/5f/47ab0d9d843c5be0f5c5bd891736a4c84fa45c3b0a0ddb6b6df7c098c66f/plotly-4.9.0-py2.py3-none-any.whl
Collecting dtreeviz==0.8.1 (from simba-uw-tf-dev)
Collecting seaborn==0.9.0 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/a8/76/220ba4420459d9c4c9c9587c6ce607bf56c25b3d3d2de62056efe482dadc/seaborn-0.9.0-py3-none-any.whl
Collecting xlrd==1.2.0 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/b0/16/63576a1a001752e34bf8ea62e367997530dc553b689356b9879339cf45a4/xlrd-1.2.0-py2.py3-none-any.whl
Collecting tables==3.6.1 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/50/e2/c26a7eec516ef3cb337e77fb8d2310d095750c634de27c295dc3466d9d02/tables-3.6.1-2-cp36-cp36m-win_amd64.whl
Collecting dash-colorscales==0.0.4 (from simba-uw-tf-dev)
Collecting shapely==1.7 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/b8/0b/e5b073f74d8c752e85d11769572f8af134d1d3c9234a8fae0a7c0d1e224a/Shapely-1.7.0-cp36-cp36m-win_amd64.whl
Collecting shap==0.35.0 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/e9/dd/1464558bff508aaddebeb6f970e6aab5b148911cea44f882822d33156250/shap-0.35.0-cp36-cp36m-win_amd64.whl
Collecting imutils==0.5.2 (from simba-uw-tf-dev)
Collecting imgaug==0.4.0 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/66/b1/af3142c4a85cba6da9f4ebb5ff4e21e2616309552caca5e8acefe9840622/imgaug-0.4.0-py2.py3-none-any.whl
Collecting cefpython3==66.0 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/af/90/7e471ba5ccbb716b42d0c1156337c9f4add8c7b83366b50ef7b7c33911da/cefpython3-66.0-py2.py3-none-win_amd64.whl
Collecting dash-html-components==1.0.3 (from simba-uw-tf-dev)
Collecting matplotlib==3.0.3 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/21/4c/35fa1837a705f33621604a1967b1505bd3f695940fdf02fad77ef11de196/matplotlib-3.0.3-cp36-cp36m-win_amd64.whl
Collecting scikit-image==0.14.2 (from simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/bd/a5/1d00f4bfe5ac7b48476b68f5c6bd87764a48c26e711f2bc510ea61d9c548/scikit_image-0.14.2-cp36-none-win_amd64.whl
Collecting dash-core-components==1.10.2 (from simba-uw-tf-dev)
Collecting Flask>=1.0.2 (from dash==1.14.0->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/cd/77/59df23681f4fd19b7cbbb5e92484d46ad587554f5d490f33ef907e456132/Flask-2.0.3-py3-none-any.whl
Collecting future (from dash==1.14.0->simba-uw-tf-dev)
Collecting dash-renderer==1.6.0 (from dash==1.14.0->simba-uw-tf-dev)
Collecting dash-table==4.9.0 (from dash==1.14.0->simba-uw-tf-dev)
Collecting flask-compress (from dash==1.14.0->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/d5/08/91ffc9506cb569f45cc4781ac8d35c6be19b9c3c8b1f8bc77aed56a46b53/Flask_Compress-1.12-py3-none-any.whl
Collecting six (from wxpython==4.0.4->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/d9/5a/e7c31adbe875f2abbb91bd84cf2dc52d792b5a01506781dbcf25c91daf11/six-1.16.0-py2.py3-none-any.whl
Collecting llvmlite<0.32.0,>=0.31.0dev0 (from numba==0.48.0->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/41/f7/1aff7e7bd1fbfe036b1274db43006162db95df2762a831480fa85ee294ed/llvmlite-0.31.0-cp36-cp36m-win_amd64.whl
Requirement already satisfied: setuptools in c:\tools\anaconda3\envs\simbaenv\lib\site-packages (from numba==0.48.0->simba-uw-tf-dev) (58.0.4)
Collecting cycler>=0.10.0 (from yellowbrick==0.9.1->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/5c/f9/695d6bedebd747e5eb0fe8fad57b72fdf25411273a39791cde838d5a8f51/cycler-0.11.0-py3-none-any.whl
Collecting jinja2 (from eli5==0.10.1->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/20/9a/e5d9ec41927401e41aea8af6d16e78b5e612bca4699d417f646a9610a076/Jinja2-3.0.3-py3-none-any.whl
Collecting attrs>16.0.0 (from eli5==0.10.1->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/be/be/7abce643bfdf8ca01c48afa2ddf8308c2308b0c3b239a44e57d020afa0ef/attrs-21.4.0-py2.py3-none-any.whl
Collecting pytz>=2017.2 (from pandas==0.25.3->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/60/2e/dec1cc18c51b8df33c7c4d0a321b084cf38e1733b98f9d15018880fb4970/pytz-2022.1-py2.py3-none-any.whl
Collecting python-dateutil>=2.6.1 (from pandas==0.25.3->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/36/7a/87837f39d0296e723bb9b62bbb257d0355c7f6128853c78955f57342a56d/python_dateutil-2.8.2-py2.py3-none-any.whl
Collecting imbalanced-learn (from imblearn==0.0->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/19/79/e86c8fd859dca4fb1fbfc61376afc63210177a235a7bfbe7219b02edf8f3/imbalanced_learn-0.9.1-py3-none-any.whl
Collecting joblib>=0.11 (from scikit-learn==0.22.2->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/3e/d5/0163eb0cfa0b673aa4fe1cd3ea9d8a81ea0f32e50807b0c295871e4aab2e/joblib-1.1.0-py2.py3-none-any.whl
Collecting retrying>=1.3.3 (from plotly==4.9.0->simba-uw-tf-dev)
Collecting colour (from dtreeviz==0.8.1->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/74/46/e81907704ab203206769dee1385dc77e1407576ff8f50a0681d0a6b541be/colour-0.1.5-py2.py3-none-any.whl
Collecting imageio (from imgaug==0.4.0->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/58/fc/1547b93534279bbf2de88f7c4a88975a65cecafd32c2bd3c518b2054ef76/imageio-2.16.0-py3-none-any.whl
Collecting kiwisolver>=1.0.1 (from matplotlib==3.0.3->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/6e/df/1250c32ab3b532c32a7e47c1cd240faba98f75b1b5150939b10e9bffb758/kiwisolver-1.3.1-cp36-cp36m-win_amd64.whl
Collecting pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.1 (from matplotlib==3.0.3->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/6c/10/a7d0fa5baea8fe7b50f448ab742f26f52b80bfca85ac2be9d35cdd9a3246/pyparsing-3.0.9-py3-none-any.whl
Collecting dask[array]>=1.0.0 (from scikit-image==0.14.2->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/2e/86/95faa4a9c1f7fbfa2df2ae9e7e1a11349cb97a81e2f38ff9dda301606882/dask-2021.3.0-py3-none-any.whl
Collecting PyWavelets>=0.4.0 (from scikit-image==0.14.2->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/30/9f/60c3b80bcefc7e3cbc76c0925e05159312cae0f3e8bf822cf50ba30b5312/PyWavelets-1.1.1-cp36-cp36m-win_amd64.whl
Collecting networkx>=1.8 (from scikit-image==0.14.2->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/f3/b7/c7f488101c0bb5e4178f3cde416004280fd40262433496830de8a8c21613/networkx-2.5.1-py3-none-any.whl
Collecting cloudpickle>=0.2.1 (from scikit-image==0.14.2->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/25/40/2c9db9cfb85a8a21c61528f6660c47662b3e59576efac610d8268d47abba/cloudpickle-2.1.0-py3-none-any.whl
Collecting Werkzeug>=2.0 (from Flask>=1.0.2->dash==1.14.0->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/f4/f3/22afbdb20cc4654b10c98043414a14057cd27fdba9d4ae61cea596000ba2/Werkzeug-2.0.3-py3-none-any.whl
Collecting click>=7.1.2 (from Flask>=1.0.2->dash==1.14.0->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/4a/a8/0b2ced25639fb20cc1c9784de90a8c25f9504a7f18cd8b5397bd61696d7d/click-8.0.4-py3-none-any.whl
Collecting itsdangerous>=2.0 (from Flask>=1.0.2->dash==1.14.0->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/9c/96/26f935afba9cd6140216da5add223a0c465b99d0f112b68a4ca426441019/itsdangerous-2.0.1-py3-none-any.whl
Collecting brotli (from flask-compress->dash==1.14.0->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/e3/d4/e68c4f2a21cab38ba1defbd5bc6ab7e74e2234383f1daf36153dce11ea9d/Brotli-1.0.9-cp36-cp36m-win_amd64.whl
Collecting MarkupSafe>=2.0 (from jinja2->eli5==0.10.1->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/73/60/296031f365b3ae96732225203d864fac7b83a185ed1820c1c87b78e154bc/MarkupSafe-2.0.1-cp36-cp36m-win_amd64.whl
Collecting threadpoolctl>=2.0.0 (from imbalanced-learn->imblearn==0.0->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/61/cf/6e354304bcb9c6413c4e02a747b600061c21d38ba51e7e544ac7bc66aecc/threadpoolctl-3.1.0-py3-none-any.whl
Collecting toolz>=0.8.2; extra == "array" (from dask[array]>=1.0.0->scikit-image==0.14.2->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/b5/f1/3df506b493736e3ee11fc1a3c2de8014a55f025d830a71bb499acc049a2c/toolz-0.11.2-py3-none-any.whl
Collecting decorator<5,>=4.3 (from networkx>=1.8->scikit-image==0.14.2->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/ed/1b/72a1821152d07cf1d8b6fce298aeb06a7eb90f4d6d41acec9861e7cc6df0/decorator-4.4.2-py2.py3-none-any.whl
Collecting dataclasses; python_version < "3.7" (from Werkzeug>=2.0->Flask>=1.0.2->dash==1.14.0->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/fe/ca/75fac5856ab5cfa51bbbcefa250182e50441074fdc3f803f6e76451fab43/dataclasses-0.8-py3-none-any.whl
Collecting importlib-metadata; python_version < "3.8" (from click>=7.1.2->Flask>=1.0.2->dash==1.14.0->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/a0/a1/b153a0a4caf7a7e3f15c2cd56c7702e2cf3d89b1b359d1f1c5e59d68f4ce/importlib_metadata-4.8.3-py3-none-any.whl
Collecting colorama; platform_system == "Windows" (from click>=7.1.2->Flask>=1.0.2->dash==1.14.0->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/44/98/5b86278fbbf250d239ae0ecb724f8572af1c91f4a11edf4d36a206189440/colorama-0.4.4-py2.py3-none-any.whl
Collecting zipp>=0.5 (from importlib-metadata; python_version < "3.8"->click>=7.1.2->Flask>=1.0.2->dash==1.14.0->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/bd/df/d4a4974a3e3957fd1c1fa3082366d7fff6e428ddb55f074bf64876f8e8ad/zipp-3.6.0-py3-none-any.whl
Collecting typing-extensions>=3.6.4; python_version < "3.8" (from importlib-metadata; python_version < "3.8"->click>=7.1.2->Flask>=1.0.2->dash==1.14.0->simba-uw-tf-dev)
  Using cached https://files.pythonhosted.org/packages/45/6b/44f7f8f1e110027cf88956b59f2fad776cca7e1704396d043f89effd3a0e/typing_extensions-4.1.1-py3-none-any.whl
imbalanced-learn 0.9.1 has requirement scikit-learn>=1.1.0, but you'll have scikit-learn 0.22.2 which is incompatible.
imbalanced-learn 0.9.1 has requirement scipy>=1.3.2, but you'll have scipy 1.1.0 which is incompatible.
imageio 2.16.0 has requirement numpy>=1.20.0, but you'll have numpy 1.18.1 which is incompatible.
imageio 2.16.0 has requirement pillow>=8.3.2, but you'll have pillow 5.4.1 which is incompatible.
Installing collected packages: six, retrying, plotly, dash-html-components, dataclasses, Werkzeug, MarkupSafe, jinja2, zipp, typing-extensions, importlib-metadata, colorama, click, itsdangerous, Flask, future, dash-renderer, dash-table, dash-core-components, brotli, flask-compress, dash, numpy, pyarrow, Pillow, wxpython, opencv-python, tqdm, tabulate, llvmlite, numba, numexpr, pyyaml, cycler, python-dateutil, kiwisolver, pyparsing, matplotlib, scipy, joblib, scikit-learn, yellowbrick, attrs, graphviz, eli5, pytz, pandas, dash-color-picker, threadpoolctl, imbalanced-learn, imblearn, h5py, xgboost, statsmodels, colour, dtreeviz, seaborn, xlrd, tables, dash-colorscales, shapely, shap, imutils, toolz, dask, PyWavelets, decorator, networkx, cloudpickle, scikit-image, imageio, imgaug, cefpython3, simba-uw-tf-dev
Successfully installed Flask-2.0.3 MarkupSafe-2.0.1 Pillow-5.4.1 PyWavelets-1.1.1 Werkzeug-2.0.3 attrs-21.4.0 brotli-1.0.9 cefpython3-66.0 click-8.0.4 cloudpickle-2.1.0 colorama-0.4.4 colour-0.1.5 cycler-0.11.0 dash-1.14.0 dash-color-picker-0.0.1 dash-colorscales-0.0.4 dash-core-components-1.10.2 dash-html-components-1.0.3 dash-renderer-1.6.0 dash-table-4.9.0 dask-2021.3.0 dataclasses-0.8 decorator-4.4.2 dtreeviz-0.8.1 eli5-0.10.1 flask-compress-1.12 future-0.18.2 graphviz-0.11 h5py-2.9.0 imageio-2.16.0 imbalanced-learn-0.9.1 imblearn-0.0 imgaug-0.4.0 importlib-metadata-4.8.3 imutils-0.5.2 itsdangerous-2.0.1 jinja2-3.0.3 joblib-1.1.0 kiwisolver-1.3.1 llvmlite-0.31.0 matplotlib-3.0.3 networkx-2.5.1 numba-0.48.0 numexpr-2.6.9 numpy-1.18.1 opencv-python-3.4.5.20 pandas-0.25.3 plotly-4.9.0 pyarrow-0.17.1 pyparsing-3.0.9 python-dateutil-2.8.2 pytz-2022.1 pyyaml-5.3.1 retrying-1.3.3 scikit-image-0.14.2 scikit-learn-0.22.2 scipy-1.1.0 seaborn-0.9.0 shap-0.35.0 shapely-1.7.0 simba-uw-tf-dev-0.91.9 six-1.16.0 statsmodels-0.9.0 tables-3.6.1 tabulate-0.8.3 threadpoolctl-3.1.0 toolz-0.11.2 tqdm-4.30.0 typing-extensions-4.1.1 wxpython-4.0.4 xgboost-0.90 xlrd-1.2.0 yellowbrick-0.9.1 zipp-3.6.0
You are using pip version 19.0.1, however version 21.3.1 is available.
You should consider upgrading via the 'python -m pip install --upgrade pip' command.

(simbaenv) C:\Users\maf315>pip uninstall shapely
Uninstalling Shapely-1.7.0:
  Would remove:
    c:\tools\anaconda3\envs\simbaenv\lib\site-packages\shapely-1.7.0.dist-info\*
    c:\tools\anaconda3\envs\simbaenv\lib\site-packages\shapely\*
Proceed (y/n)? y
  Successfully uninstalled Shapely-1.7.0

(simbaenv) C:\Users\maf315>conda install -c conda-forge shapely
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: C:\TOOLS\Anaconda3\envs\simbaenv

  added / updated specs:
    - shapely

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    geos-3.8.0                 |       h33f27b4_0         905 KB
    intel-openmp-2022.1.0      |    h57928b3_3787         3.7 MB  conda-forge
    libblas-3.9.0              |     14_win64_mkl         5.3 MB  conda-forge
    libcblas-3.9.0             |     14_win64_mkl         5.3 MB  conda-forge
    liblapack-3.9.0            |     14_win64_mkl         5.3 MB  conda-forge
    numpy-1.19.5               |   py36h4b40d73_2         4.9 MB  conda-forge
    python_abi-3.6             |          2_cp36m           4 KB  conda-forge
    shapely-1.7.1              |   py36h06580b3_0         369 KB
    tbb-2021.5.0               |       h2d74725_1         148 KB  conda-forge
    ------------------------------------------------------------
                                           Total:        25.7 MB

The following NEW packages will be INSTALLED:

  geos               pkgs/main/win-64::geos-3.8.0-h33f27b4_0
  intel-openmp       conda-forge/win-64::intel-openmp-2022.1.0-h57928b3_3787
  libblas            conda-forge/win-64::libblas-3.9.0-14_win64_mkl
  libcblas           conda-forge/win-64::libcblas-3.9.0-14_win64_mkl
  liblapack          conda-forge/win-64::liblapack-3.9.0-14_win64_mkl
  mkl                conda-forge/win-64::mkl-2022.0.0-h0e2418a_796
  numpy              conda-forge/win-64::numpy-1.19.5-py36h4b40d73_2
  python_abi         conda-forge/win-64::python_abi-3.6-2_cp36m
  shapely            pkgs/main/win-64::shapely-1.7.1-py36h06580b3_0
  tbb                conda-forge/win-64::tbb-2021.5.0-h2d74725_1

Proceed ([y]/n)? y

Downloading and Extracting Packages
libcblas-3.9.0       | 5.3 MB    | ############################################################################ | 100%
python_abi-3.6       | 4 KB      | ############################################################################ | 100%
geos-3.8.0           | 905 KB    | ############################################################################ | 100%
tbb-2021.5.0         | 148 KB    | ############################################################################ | 100%
numpy-1.19.5         | 4.9 MB    | ############################################################################ | 100%
libblas-3.9.0        | 5.3 MB    | ############################################################################ | 100%
intel-openmp-2022.1. | 3.7 MB    | ############################################################################ | 100%
liblapack-3.9.0      | 5.3 MB    | ############################################################################ | 100%
shapely-1.7.1        | 369 KB    | ############################################################################ | 100%
Preparing transaction: done
Verifying transaction: done
Executing transaction: done

(simbaenv) C:\Users\maf315>simba
Traceback (most recent call last):
  File "c:\tools\anaconda3\envs\simbaenv\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\tools\anaconda3\envs\simbaenv\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\TOOLS\Anaconda3\envs\simbaenv\Scripts\simba.exe\__main__.py", line 5, in <module>
  File "c:\tools\anaconda3\envs\simbaenv\lib\site-packages\simba\SimBA.py", line 50, in <module>
    from simba.train_multiple_models_from_meta import *
  File "c:\tools\anaconda3\envs\simbaenv\lib\site-packages\simba\train_multiple_models_from_meta.py", line 10, in <module>
    from imblearn.combine import SMOTEENN
  File "c:\tools\anaconda3\envs\simbaenv\lib\site-packages\imblearn\__init__.py", line 53, in <module>
    from . import ensemble
  File "c:\tools\anaconda3\envs\simbaenv\lib\site-packages\imblearn\ensemble\__init__.py", line 6, in <module>
    from ._easy_ensemble import EasyEnsembleClassifier
  File "c:\tools\anaconda3\envs\simbaenv\lib\site-packages\imblearn\ensemble\_easy_ensemble.py", line 21, in <module>
    from ..pipeline import Pipeline
  File "c:\tools\anaconda3\envs\simbaenv\lib\site-packages\imblearn\pipeline.py", line 18, in <module>
    from sklearn.utils.metaestimators import available_if
ImportError: cannot import name 'available_if'

(simbaenv) C:\Users\maf315>simba
Traceback (most recent call last):
  File "c:\tools\anaconda3\envs\simbaenv\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\tools\anaconda3\envs\simbaenv\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\TOOLS\Anaconda3\envs\simbaenv\Scripts\simba.exe\__main__.py", line 5, in <module>
  File "c:\tools\anaconda3\envs\simbaenv\lib\site-packages\simba\SimBA.py", line 50, in <module>
    from simba.train_multiple_models_from_meta import *
  File "c:\tools\anaconda3\envs\simbaenv\lib\site-packages\simba\train_multiple_models_from_meta.py", line 10, in <module>
    from imblearn.combine import SMOTEENN
  File "c:\tools\anaconda3\envs\simbaenv\lib\site-packages\imblearn\__init__.py", line 53, in <module>
    from . import ensemble
  File "c:\tools\anaconda3\envs\simbaenv\lib\site-packages\imblearn\ensemble\__init__.py", line 6, in <module>
    from ._easy_ensemble import EasyEnsembleClassifier
  File "c:\tools\anaconda3\envs\simbaenv\lib\site-packages\imblearn\ensemble\_easy_ensemble.py", line 21, in <module>
    from ..pipeline import Pipeline
  File "c:\tools\anaconda3\envs\simbaenv\lib\site-packages\imblearn\pipeline.py", line 18, in <module>
    from sklearn.utils.metaestimators import available_if
ImportError: cannot import name 'available_if'

___________________________________________________________________________________________

**Expected behavior**
I expected to follow the installation steps and for the software to open successfully.

**Screenshots**
<img width="934" alt="ffmpeg issue" src="https://user-images.githubusercontent.com/99504167/171903462-cf27bc0a-d9c4-4ea6-a25b-18de23f18252.png">
<img width="554" alt="simBA installation directions" src="https://user-images.githubusercontent.com/99504167/171903467-2a9f315b-9f78-434a-b79d-3a4e1fb2b7e7.png">
<img width="617" alt="step 6 of ffmpeg installation process" src="https://user-images.githubusercontent.com/99504167/171903468-43a96e36-e90c-4f7b-b705-a42fd1b0170f.png">
<img width="646" alt="steps for installation" src="https://user-images.githubusercontent.com/99504167/171903469-26c751bd-cf74-4374-ac5a-ce7c822ea6f5.png">

**Desktop (Lenovo P340 Workstation (ThinkStation) - Type 30DH (Tower Form Factor)):**
 - OS build: 19043.1706 (Processor: Intel(R) Core(TM) i9-10900K CPU @ 3.70GHz   3.70 GHz)
 - Python Version 3.10.4
 - Are you using anaconda? - Yes
 

**Additional context**
I am working through a remote desktop while using this software.

**Comment by sronilsson** [comment]
Hi @fon215 ! Thanks for reporting - I had a similar issue reported very recently by @m-migliaro. The solution for them to **not** downgrade pip to version `19.0.1`, and stick with version `21.2.2`. Can you check if this also works for you?

**Comment by fon215** [comment]
That worked! Thank you so much!

**Comment by fon215** [comment]
> Hi @fon215 ! Thanks for reporting - I had a similar issue reported very recently by @m-migliaro. The solution for them to **not** downgrade pip to version `19.0.1`, and stick with version `21.2.2`. Can you check if this also works for you?

That helped me open SimBA, however, I am still having trouble installing FFmpeg. Could someone direct me on getting past the sixth step listed in the screenshot I provided?

**Comment by sronilsson** [comment]
Hi @fon215 - I think you need to install a program that allows you to uncompress the 7Zip file, either winzip, winrar, or 7-zip (https://www.7-zip.org/)

**Comment by fon215** [comment]
Okay, thank you.
